Content.Client/Administration/UI/Tabs/ObjectsTab/ObjectsTab.xaml.cs

```diff
@@ -81,32 +111,63 @@ private void RefreshObjectList(ObjectsTabSelection selection)
                 throw new ArgumentOutOfRangeException(nameof(selection), selection, null);
         }
 
-        foreach (var control in _objects)
-        {
-            ObjectList.RemoveChild(control);
-        }
+        var sortedEntities = _ascending
+            ? entities.OrderBy(e => GetComparableValue(e, _headerClicked)).ToList()
+            : entities.OrderByDescending(e => GetComparableValue(e, _headerClicked)).ToList();
 
-        _objects.Clear();
+        SearchList.PopulateList(sortedEntities.Select((info, index) =>
+            new ObjectsListData(info, $"{info.Name} {info.Entity}", index % 2 == 0 ? _altColor : _defaultColor)).ToList());
```

Comment:
populate in a loop, there shouldnt be any linq here apart from the ToList and stuff